tests/integration-tests/src/test/resources/configs/mt_keycloak.conf

```diff
@@ -31,7 +31,6 @@ roles = [
     {
         name = "Issuer"
         url = "${ISSUER_AGENT_URL:-http://localhost:8080}"
-        apikey = "${ISSUER_API_KEY:-${random.string(16)}}"
```

Comment:
Why did you remove the apikey? Isn't that being used for remote environment?

Reply:
it looks, in this spec, we don't use APIKEY authentication (JWT is used for both `tenant` and `admin`.

Reply:
For the `mt_keycloak.conf` one, I think we aren't using it. At least the sandbox environment doesn't require `apikey` header to bypass APISIX anymore.